vagrant/ansible/cluster-vars.yml

```diff
@@ -37,3 +43,10 @@ gluster_cluster_volume: vol
 gluster_cluster_replica_count: 2
 gluster_cluster_transport: 'tcp'
 gluster_cluster_bricks: '/bricks/brick0/vol,/bricks/brick1/vol'
+
+#Autogenerate these values containing ip addresses
+gluster_features_ctdb_nodes: 192.168.122.100,192.168.122.101
+gluster_features_ctdb_publicaddr: '192.168.123.10/24 eth2,192.168.123.11/24 eth2'
+gluster_features_ctdb_volume: ctdb
+
+ctdb_netbios_name: "RHGS-TEST"
```

Comment:
Let's not mention RHGS

Reply:
Ack

Reply:
Changed netbios name- 
samba_netbios_name: "GLUSTERTEST"
and we now set it in smb.conf using jinja template